src/image/PhotoCalib.cc

```diff
@@ -53,6 +53,23 @@ double toMaggiesErr(double instFlux, double instFluxErr, double scale, double sc
     return maggies * hypot(instFluxErr / instFlux, scaleErr / scale);
 }
 
+// 2,2 implies that the arrays are memory contiguous.
+// Note: does not take scale (like the above), because normal usage is to compute maggies directly,
+// without saving the calibration at each point.
```

Comment:
I've written a docstring for it and removed those other comments (the 2,2 one was from an earlier version, before I found that you 2,2 doesn't work here, because Image.getArray() returns a 2,1 view).